src/core/opamFilename.mli

```diff
@@ -322,5 +322,15 @@ module Attribute: sig
 
 end
 
+module SubPath: sig
+
+  include OpamStd.ABSTRACT
+  val (/): Dir.t -> t -> Dir.t
+  val (/?): Dir.t -> t option -> Dir.t
```

Comment:
new operators for subpaths, to help using then as in most cases it is an option given to a function